pkg/util/utils.go

```diff
@@ -723,3 +726,316 @@ func SocketPath() (string, error) {
 	// Glue the socket path together
 	return filepath.Join(xdg, "podman", "podman.sock"), nil
 }
+
+// ImageScpOptions provide options for securely copying images to and from a remote host
+type ImageScpOptions struct {
```

Comment:
Isn't this already defined above?

Reply:
this is a struct used in utils since entities cannot be imported into utils.